components/forms/admin-api-key.tsx

```diff
@@ -15,19 +23,26 @@ const initialState = {
   errors: {},
 }
 
-export function AdminApiKeyForm() {
+export function AdminApiKeyForm({ teams }: { teams: Team[] }) {
   const [state, formAction] = useFormState(generateApiKey, initialState)
 
   return (
     <form className="flex flex-col gap-4" action={formAction}>
-      <Input
-        id="email"
-        name="email"
-        type="email"
-        placeholder="User's email"
-        required
-      />
-
+      <Select name="team" required>
+        <SelectTrigger>
+          <SelectValue placeholder="Select team" />
```

Comment:
<img width="892" alt="image" src="https://github.com/user-attachments/assets/2bdb9c4f-1e77-46f1-b270-0b0f8145211f" />

Small note, we called them "Users" during the creation step... should we not continue that terminology in the front end?

```suggestion
          <SelectValue placeholder="Select user" />
```

Reply:
We use the term "prover", which I think is the more correct term for UX, at least for this type of role. Everyone is a user, but in the frontend I think we should use role names to make it clearer what we are talking about